libs/block-editor/src/lib/plugins/bubble-link-form.plugin.ts

```diff
@@ -70,6 +70,9 @@ export class BubbleLinkFormView {
 
         this.editor.on('focus', this.focusHandler);
         this.setComponentEvents();
+
+        // We do not need only listen editor scroll event but all scroll on the website.
+        document.body.addEventListener('scroll', this.hanlderScroll.bind(this), true);
```

Comment:
Did you tests this in all major browsers?

Reply:
`scroll` event has full support. 
[More Information](https://developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event#browser_compatibility_

Reply:
`scroll` event has full support.

[More Information](https://developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event#browser_compatibility)